rpcserver.go

```diff
@@ -1333,6 +1360,7 @@ var rpcHandlers = map[string]requestHandler{
 	// Reference implementation wallet methods (implemented)
 	"addmultisigaddress":     AddMultiSigAddress,
 	"createmultisig":         CreateMultiSig,
+	"createnewaccount":       CreateNewAccount,
```

Comment:
Move this (and renameaccount) to to the extensions below; there is no equivalent bitcoind request.